R/theme.R

```diff
@@ -17,28 +17,30 @@
 #' to see the full list of theming variables available
 #'
 
-default_theme <- bslib::bs_theme(
-  # principal variables in use by connectwidgets
-  bg = "#FFF",
-  fg = "#000",
-  primary = "#346899",
-  secondary = "#858585",
-  dark = "#252525",
-  light = "#EDF6FF",
-  base_font = "Lato, sans-serif",
-  heading_font = "Lato, sans-serif",
-
-  # more complementary variables
-  "font-size-base" = "1.44rem",
-  "border-color" = "#F2F2F2",
-  "gray-100" = "#F8F8F8",
-  "gray-200" = "#F2F2F2",
-  "gray-300" = "#ECECEC",
-  "gray-400" = "#CDCDCD",
-  "gray-500" = "#B0B0B0",
-  "gray-600" = "#858585",
-  "gray-700" = "#303030"
-)
+default_theme <- function() {
+  bslib::bs_theme(
+    # principal variables in use by connectwidgets
+    bg = "#FFF",
+    fg = "#000",
+    primary = "#346899",
+    secondary = "#858585",
+    dark = "#252525",
+    light = "#EDF6FF",
+    base_font = "Lato, sans-serif",
+    heading_font = "Lato, sans-serif",
```

Comment:
May also want to consider using `bslib::font_google()` to automatically import Lato font files